Wireframe/index.html

```diff
@@ -4,10 +4,45 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
+      <header>
+        <h1>GIT</h1>
+        <P>(Information Tracker)</P>
+      </header>
+      <main>
+        <article>
+          <img src="//res.cloudinary.com/practicaldev/image/fetch/s--ZqwTtpA6--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/6vq45tzju4l6m3aijeoq.png" alt="A picture with three computers and different users explaining about git">
+          <h2>What is git?</h2>
+          <p>Git is a version control system that is distributed, with the purpose of managing and tracking changes made in software projects.</p>
+          <a href="https://www.atlassian.com/git/tutorials/what-is-git"><button>Read More</button></a>
+        </article>
+        <section class="container">
+          <section class="child1">
```

Comment:
are these child `section` elements necessary as they just wrap a single `article` element

instead of 

```
<section>
  <article>
    ...
  </article>
</section>
<section>
  <article>
    ...
  </article>
</section>
```

You can just do 

```
<article>
    ....
</article>
<article>
   ....
</article>
```

this will make your markup more cleaner and easier to read.